### PR TITLE
badge

### LINKED ISSUE #94: Show badge on user profile
1. Staff
1. Teacher
1. Teacher Assistant

(Show only one badge, in that order of preference)

### PR DESCRIPTION
**Pull request by tchret**
fixes #94 

image

image

image

image

### CODE REVIEW

File: app/assets/javascripts/components/user_profile.jsx [hunk 1]
```diff
@@ -63,6 +68,8 @@ class UserProfile extends React.Component {
         </a>);
     }
 
+    console.log(this.props)
```
Comment (ssaunier) [inline app/assets/javascripts/components/user_profile.jsx]: To remove.

File: app/assets/stylesheets/components/_badge.scss [hunk 2]
```diff
@@ -11,3 +11,11 @@
     border-color: $green;
   }
 }
+
+.badge {
+  &.staff { background: #D23333 }
+  &.teacher { background: $green }
+  &.teacher { background: $orange }
```
Comment (ssaunier) [inline app/assets/stylesheets/components/_badge.scss]: deux fois la meme prop ?

**Comment by ssaunier** [comment]
@tchret : review done, waiting for next commit.

**Comment by tchret** [comment]
👍

**Pull request merged**